gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -868,6 +868,22 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
        call ESMF_ConfigGetAttribute(cfg, value=list(n)%recycle_track, default=.false., &
                                     label=trim(string) // 'recycle_track:', _RC)
 
+! Get a single station-data file containing id/name/lon/lat/elev
+       call ESMF_ConfigGetAttribute(cfg, value=list(n)%observation_spec, default="", &
+            label=trim(string) // 'observation_spec:', _RC)  
+       call ESMF_ConfigGetAttribute(cfg, value=list(n)%stationIdFile, default="", &
+            label=trim(string) // 'station_id_file:', _RC)
+       !!call ESMF_ConfigGetAttribute(cfg, value=list(n)%stationDataFile, default="", &
+       !!     label=trim(string) // 'station_data_file:', _RC)
+       !!call ESMF_ConfigGetAttribute(cfg, value=list(n)%maskFile, default="", &
+       !!                             label=trim(string) // 'mask_file:', _RC)
+       !!call ESMF_ConfigGetAttribute(cfg, value=list(n)%trajectoryFile, default="", &
+       !!                             label=trim(string) // 'trajectory_file:', _RC)
+       !!call ESMF_ConfigGetAttribute(cfg, value=list(n)%swathFile, default="", &
+       !!                             label=trim(string) // 'swath_file:', _RC)       
+       write(6,'(3(x,a,x))') 'list(n)%observation_spec, list(n)%stationIdFile:', &
+            trim(list(n)%observation_spec), trim(list(n)%stationIdFile)
+       
```

Comment:
This bit looks useful, but should be emitted by a call to `lgr%debug()`.       You should be able to find examples elsewhere in MAPL.  (Based on python `logger` package.)

This will allow us to throttle such outputs in production runs and turn them on when we have an issue.